franklin/github/views.py

```diff
@@ -209,8 +148,24 @@ def repository_detail(request, pk):
         serializer = SiteSerializer(site)
         return Response(serializer.data)
     elif request.method == 'DELETE':
-        site.delete()
-        return Response(status=status.HTTP_204_NO_CONTENT)
+        site.is_active = False
+        site.save()
+        return delete_site(site, request.user)
```

Comment:
set `is_active` to false and put the site on the chopping block......which currently happens immediately.